package.json

```diff
@@ -31,23 +31,21 @@
     "hide",
     "header",
     "styled-components",
-    "typescript",
-    "flow"
+    "typescript"
   ],
-  "module": "index.js",
+  "module": "./index.js",
   "scripts": {
-    "build": "node tools/build.js",
-    "build:demo": "node tools/build.demo.js",
+    "build": "ts-node tools/build.ts",
+    "build:demo": "ts-node tools/build.demo.ts",
     "test": "jest --config jest.config.json",
     "test:coverage": "jest --config jest.config.json --coverage",
     "test:watch": "jest --config jest.config.json --watchAll",
     "test:update": "jest --config jest.config.json -u",
     "lint": "npm run eslint && npm run stylelint",
     "lint:fix": "npm run eslint --fix && npm run stylelint",
     "eslint": "eslint .",
-    "stylelint": "stylelint './src/**/*.js'",
-    "flow:check-now": "flow check",
-    "flow:install-types": "flow-typed install --skip",
+    "stylelint": "stylelint './src/**/*.{js,ts,tsx}'",
+    "ts:build": "tsc --build",
```

Comment:
What's the difference between "build" and "ts:build"? Do we need both?

Reply:
I renamed ts:build with `ts:check` to make clear that this just checks the typescript types.